plugins/rtmp-services/data/services.json

```diff
@@ -2846,6 +2846,262 @@
             "supported video codecs": [
                 "h264"
             ]
+        },
+        {
+            "name": "Wavezoo SD",
+            "common": true,
+            "stream_key_link": "https://wavezoo.com",
+            "supported video codecs": [
+                "h264"
+            ],
+            "supported audio codecs": [
+                "aac"
+            ],
+            "servers": [
+                {
+                    "name": "Server #01 (RTMP)",
+                    "url": "rtmp://live-stream-server-01:1935/LiveApp"
```

Comment:
Is this hostname (`live-stream-server-01`) correct? You might need to add your domain name, which is automatically added when you attempt to resolve the hostname.